<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>tt3::gui::AskYesNoDialog</class>
 <widget class="QDialog" name="tt3::gui::AskYesNoDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>484</width>
    <height>121</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="promptLabel">
     <property name="text">
      <string>&lt;PROMPT&gt;</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QCheckBox" name="assumeYesCheckBox">
     <property name="text">
      <string>Do not request this confirmation again (assume the answer is yes)</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::StandardButton::No|QDialogButtonBox::StandardButton::Yes</set>
     </property>
     <property name="centerButtons">
      <bool>false</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>tt3::gui::AskYesNoDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>241</x>
     <y>95</y>
    </hint>
    <hint type="destinationlabel">
     <x>241</x>
     <y>60</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>tt3::gui::AskYesNoDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>241</x>
     <y>95</y>
    </hint>
    <hint type="destinationlabel">
     <x>241</x>
     <y>60</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
